pom.xml

```diff
@@ -123,6 +124,39 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- 3. Convert java datmodel to typescript datamodel -->
```

Comment:
Looks Ok for me at the first view. After a checkout and a local build i will give my approval if it works.